pages/help_dialog.coffee

```diff
@@ -38,6 +38,16 @@ HelpDialog =
 
     @showAdvancedCommands(@getShowAdvancedCommands())
 
+    # When command names are shown, clicking on them copies their text to the clipboard (and they can be
+    # clicked with link hints).
+    for element in @dialogElement.getElementsByClassName "commandName"
+      do (element) ->
+        element.classList.add "vimiumClickable"
```

Comment:
Why not use the actual standard way to mark the element as a link (`element.setAttribute "role", " link"`) rather than create a new one?